src/ECS/Systems/Implementations/DynamicsSystem.cpp

```diff
@@ -7,6 +7,8 @@
  * openblack is licensed under the GNU General Public License version 3.
  *****************************************************************************/
 
+#define LOCATOR_IMPLEMENTATIONS 1
```

Comment:
You don't need the 1.
`#define LOCATOR_IMPLEMENTATIONS ` is enough for `#if !defined(LOCATOR_IMPLEMENTATIONS)`